docs/mcp.md

```
@@ -0,0 +1,88 @@
+
+=== "Docker"
+
+    ```bash
+    docker run -d -p 8080:8080 -p 2525:2525 \
+      -e EnableMcpServer=true \
+      changemakerstudiosus/papercut-smtp:latest
+    ```
+
+Restart the service after changing it. When enabled, the endpoint is served at:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)